src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -93,12 +93,19 @@ static void compoundObjectResponse(orion::CompoundValueNode* cvP, const BSONElem
 *
 * mongoConnect -
 */
-bool mongoConnect(const char* host, const char* db, const char* rplSet, const char* username, const char* passwd, bool _multitenant) {
+bool mongoConnect(const char* host,
+                  const char* db,
+                  const char* rplSet,
+                  const char* username,
+                  const char* passwd,
+                  bool        _multitenant,
+                  double      timeout)
+{
 
     std::string err;
     multitenant = _multitenant;
 
-    LM_T(LmtBug, ("dbName: '%s'", db));
+    LM_T(LmtBug, ("dbName: '%s', rplSet: '%s', timeout: %f", db, rplSet, timeout));
```

Comment:
Strange trace level for a normal trace ...
This tracelevel is probably meant for buggy situations.

Stupid example to illustrate what I mean:

    bool X;

    switch (X)
    {
    case true:  break;
    case false: break;
    default:      LM_T(LmtBug, ("How did we get here?")); break;
    }